README.md

```diff
@@ -603,6 +603,72 @@ Result:
 path/to/file.png?e43b20c069c4a01867c31e98cbce33c9
 ```
 
+---
```

Comment:
Let's use heading here instead `---`, something like `### Dynamic public path with __webpack_public_path__` (maybe better name for heading)

Reply:
i added a little paragraph for explaining the use case. makes sense?